km_api/know_me/tests/models/test_km_user_accessor_model.py

```diff
@@ -20,4 +20,5 @@ def test_string_conversion(km_user_accessor_factory):
     """
     accessor = km_user_accessor_factory()
 
-    assert str(accessor) == "Accessor for " + accessor.km_user.name
+    expected = 'Accessor for {user}'.format(user=accessor.km_user.name)
```

Comment:
I'm not sure I follow. The Know Me user's name property and the string representation of the Know Me user currently return the same thing, and I can't find a to string method in the User model. What am I missing?